compiler/dcalc/from_scopelang.ml

```diff
@@ -135,7 +135,10 @@ let tag_with_log_entry
     (l : log_entry)
     (markings : Uid.MarkedString.info list) : 'm Ast.expr boxed =
   let m = mark_tany (Marked.get_mark e) (Expr.pos e) in
-  Expr.eapp (Expr.eop (Log (l, markings)) [TAny, Expr.pos e] m) [e] m
+
+  if !Cli.trace_flag then
```

Comment:
we need to add the incompatiblity between the trace flag and the avoid_exception flag.